.gitlab-ci-check-license.yml

```diff
@@ -34,6 +34,9 @@ test:check-license:
     - mender-qa-worker-generic-light
   stage: test
   needs: []
+  rules:
+    - exists:
+        - LIC_FILES_CHKSUM.sha256
```

Comment:
Suggestion: Maybe this should be a negative match for `licenses.csv` instead, just to make sure we don't accidentally omit a job for any repository (particularly newly created ones).

Reply:
I agree with this :+1: